Library/Homebrew/dev-cmd/create.rb

```diff
@@ -136,6 +137,8 @@ def create
 
     fc.generate!
 
+    PyPI.update_python_resources! Formula[fc.name], ignore_non_pypi_packages: true if args.python?
```

Comment:
Will this fail if `pipgrip` is not installed?

Reply:
Yep. Thanks!